docs/BR.md

```diff
@@ -853,30 +853,26 @@ The CA SHALL update and reissue CRLs at least (i) once every twelve months and (
 No stipulation.
 
 ### 4.9.9 On-line revocation/status checking availability
-OCSP responses MUST conform to RFC6960 and/or RFC5019. OCSP responses MUST either:
-
-1. Be signed by the CA that issued the Certificates whose revocation status is being checked, or
-2. Be signed by an OCSP Responder whose Certificate is signed by the CA that issued the Certificate whose
-revocation status is being checked.
-
-In the latter case, the OCSP signing Certificate MUST contain an extension of type id-pkix-ocsp-nocheck, as
-defined by RFC6960.
+The CA SHALL provide an OCSP responder that conforms to the profile specified by RFC5019 for all Certificates issued, including the transport profile.
```

Comment:
> This is a refinement on the requirement from 866/867

What does 866/867 refer to here?

> It is more restrictive than the current language, which permits RFC 6960 A.1 style GETs, but matches what Clients and Servers are already expecting (notably, IIS).

By my reading, the only way in which [RFC5019 GETs](https://tools.ietf.org/html/rfc5019#section-5) are more restrictive than [RFC 6960 GETs](https://tools.ietf.org/html/rfc6960#appendix-A.1) is that they specify some response headers. Is that accurate?